### ISSUE TITLE
Missing aws-sdk when performing a deploy

### ISSUE THREAD

**Issue by adriangmweb** [boostercloud/booster#510]
# Bug Report

## Current Behavior

After running the deploy command `boost deploy -e production`, it fails and throws the following error:
```
❯ boost deploy -e production
    Error: Error: Cannot find module 'aws-sdk/clients/dynamodb'
    Require stack:
    - /Users/adriangonzalezmesa/Projects/Booster/demo-adriangm/node_modules/@boostercloud/framework-provider-aws/dist/library/events-adapter.js
    - /Users/adriangonzalezmesa/Projects/Booster/demo-adriangm/node_modules/@boostercloud/framework-provider-aws/dist/index.js
    - /Users/adriangonzalezmesa/Projects/Booster/demo-adriangm/dist/config/config.js
    - /Users/adriangonzalezmesa/Projects/Booster/demo-adriangm/node_modules/@boostercloud/framework-core/dist/importer.js
    - /Users/adriangonzalezmesa/Projects/Booster/demo-adriangm/node_modules/@boostercloud/framework-core/dist/booster.js
    - /Users/adriangonzalezmesa/Projects/Booster/demo-adriangm/node_modules/@boostercloud/framework-core/dist/index.js
    - /Users/adriangonzalezmesa/Projects/Booster/demo-adriangm/dist/index.js
    - /Users/adriangonzalezmesa/.nvm/versions/node/v12.20.0/lib/node_modules/@boostercloud/cli/dist/services/config-service.js
    - /Users/adriangonzalezmesa/.nvm/versions/node/v12.20.0/lib/node_modules/@boostercloud/cli/dist/commands/deploy.js
    - /Users/adriangonzalezmesa/.nvm/versions/node/v12.20.0/lib/node_modules/@boostercloud/cli/node_modules/@oclif/config/lib/plugin.js
    - /Users/adriangonzalezmesa/.nvm/versions/node/v12.20.0/lib/node_modules/@boostercloud/cli/node_modules/@oclif/config/lib/config.js
    - /Users/adriangonzalezmesa/.nvm/versions/node/v12.20.0/lib/node_modules/@boostercloud/cli/node_modules/@oclif/config/lib/index.js
    - /Users/adriangonzalezmesa/.nvm/versions/node/v12.20.0/lib/node_modules/@boostercloud/cli/node_modules/@oclif/command/lib/command.js
    - /Users/adriangonzalezmesa/.nvm/versions/node/v12.20.0/lib/node_modules/@boostercloud/cli/node_modules/@oclif/command/lib/index.js
    - /Users/adriangonzalezmesa/.nvm/versions/node/v12.20.0/lib/node_modules/@boostercloud/cli/bin/run
        at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
        at Function.Module._load (internal/modules/cjs/loader.js:667:27)
        at Module.require (internal/modules/cjs/loader.js:887:19)
        at require (internal/modules/cjs/helpers.js:74:18)
        at Object.<anonymous> 
    (/Users/adriangonzalezmesa/Projects/Booster/demo-adriangm/node_modules/@boostercloud/framework-provider-aws/dist/library/events-adapter.js:6:20)
        at Module._compile (internal/modules/cjs/loader.js:999:30)
        at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
        at Module.load (internal/modules/cjs/loader.js:863:32)
        at Function.Module._load (internal/modules/cjs/loader.js:708:14)
        at Module.require (internal/modules/cjs/loader.js:887:19)
```

## Expected behavior

The app should be deployed to AWS without any errors.

## Possible Solution

Adding back the `framework-provider-aws-infrastructure` package to dependencies or find a way of using the `aws-sdk` package without uploading into AWS.

### Environment

- Booster version: .0.9.3
- Node/npm version: v12.20.0
- OS: OSX 11.1 Big Sur

**Comment by javiertoledo** [comment]
Assigning this to me because this should be solved after #530 is merged

**Comment by davidverdu** [comment]
I temporally solved the issue by running npm install aws-sdk inside the sample project. This won't remain as the final solution, since a mayor fix is ongoing
